src/LB/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM haproxytech/haproxy-ubuntu:2.0
```

Comment:
I don't think that LB is not a good name for this folder. It also holds the Dockerfile with Mongo for example.
Let's rename the folder to Infrastructure and move all DevOps-related files here.

Reply:
done